packages/grafana-ui/src/components/Slider/Slider.story.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Slider } from './Slider';
+import { select, number, boolean } from '@storybook/addon-knobs';
+
+const getStory = (title: string, component: any) => ({
```

Comment:
Not sure why you need this?  I think introducing such function should happen in `utils/storybook` and should allow some more customisation of the stories, like `parameters` customisation.

In this case default exporting an object should be enough.

Reply:
Yes, you are right, we don't need it atm. I  was thinking, that if we add storybook slider docs in the future, it would be ready for it, but now it is redundant. So I've removed it.